cms/djangoapps/contentstore/views/course.py

```diff
@@ -744,6 +760,8 @@ def settings_handler(request, course_key_string):
         json: update the Course and About xblocks through the CourseDetails model
     """
     course_key = CourseKey.from_string(course_key_string)
+    prerequisite_course_enabled = settings.FEATURES.get('ENABLE_PREREQUISITE_COURSES', False)
+    milestones_app_enabled = settings.FEATURES.get('MILESTONES_APP', False)
```

Comment:
I'd suggest a hard assert. Better to let operators know that something is misconfigured rather than some obscure error happening.